react/week1/todo-list/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const toDos = [
+  {
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017"
+  },
+  {
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017"
+  },
+  {
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017"
+  }
+
+]
+
+function Header () {
+  return (
+    <h1>Todo List</h1>
+  )
+}
+
+function ListItems (props) {
+  return (
+    <li>
+      {props.description} : {props.deadline}
+    </li>
+    
+  )
+};
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <ol> 
+        {toDos.map(toDo => {
+          return (<ListItems description = {toDo.description} deadline = {toDo.deadline} />)
```

Comment:
try to work a bit with formatting here. for example

    return (
      <ListItems description={toDo.description} deadline={toDo.deadline} />
    );

is a little easier to read